src/borg/testsuite/platform_freebsd.py

```diff
@@ -0,0 +1,25 @@
+""" Dummy file for now, will eventually contain freebsd ACL tests """
+import pytest
+
+from .platform import skipif_not_freebsd
+
+# set module-level skips
+pytestmark = [skipif_not_freebsd]
+
+
+def get_acl():
+    return
+
+
+def get_set_acl():
+    return
+
+
+@pytest.mark.skip(reason="not yet implemented")
+def test_access_acl():
+    pass
+
+
+@pytest.mark.skip(reason="not yet implemented")
+def test_default_acl():
+    pass
```

Comment:
additionally to access and default, freebsd also has `nfs4` acls.

other than that: yeah, that's what I had in mind. like this, it is more obvious that there is something missing and maybe it motivates some developer to add that.

Reply:
Excellent, I have added that additional test and just committed the changes.